run_server.sh

```diff
@@ -17,11 +17,23 @@
 TEAM_ID="$1"
 TEAM_SECRET="$2"
 PORT="$3"
+RELAY_ADDRESS="$4"
 
 if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" ]] ; then
-  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT>'
+  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> [RELAY ADDRESS]'
   exit 1
 fi
 
 cd './bin'
-java codeu.chat.ServerMain "$TEAM_ID" "$TEAM_SECRET" "$PORT"
+if [ "$RELAY_ADDRESS" == "" ] ; then
```

Comment:
I don't think so. You gave it a parameter in the spot that is suppose to the relay address. Giving it an empty string is just as invalid as giving it garbage.